src/main/webapp/app/exercises/quiz/manage/quiz-exercise.component.html

```diff
@@ -142,18 +142,22 @@ <h4 class="d-flex mb-3 flex-wrap justify-content-end">
                                 </div>
                             </ng-container>
                             <button
-                                id="quiz-set-end-{{ quizExercise.id }}"
-                                type="submit"
-                                (click)="endQuiz(quizExercise.id!)"
-                                class="btn btn-warning btn-sm me-1 mb-1"
                                 *ngIf="
                                     (quizExercise.status === QuizStatus.VISIBLE || quizExercise.status === QuizStatus.ACTIVE) &&
                                     quizExercise.quizMode !== QuizMode.SYNCHRONIZED &&
-                                    quizExercise.isAtLeastEditor
+                                    quizExercise.isAtLeastInstructor
                                 "
+                                id="quiz-set-end-{{ quizExercise.id }}"
+                                class="mb-1"
+                                jhiDeleteButton
+                                [actionType]="ActionType.EndNow"
+                                [entityTitle]="quizExercise.title || ''"
+                                deleteQuestion="artemisApp.quizExercise.endNow.question"
+                                deleteConfirmationText="artemisApp.exercise.delete.typeNameToConfirm"
+                                (delete)="endQuiz(quizExercise.id!)"
+                                [dialogError]="dialogError$"
                             >
                                 <fa-icon [icon]="faStopCircle"></fa-icon>
```

Comment:
Although the color used is a good choice, to me the icon used for this button does not stress the severity of the situation enough. How about using the `fa-triangle-exclamation` icon?

```suggestion
                                <fa-icon [icon]="faTriangleExclamation"></fa-icon>
```

Reply:
![image](https://user-images.githubusercontent.com/67344158/168373720-4bf506c1-b94e-4943-88f0-c996ea0fb9db.png)

Reply:
I don't think that action is actually that dangerous with the confirmation (i may have been somewhat annoyed when i wrote the testing steps) and i don't think the symbol fits with the action of the button 
![image](https://user-images.githubusercontent.com/44437514/168374777-f92a0753-f6ad-4e3e-9752-7c7f8e6e8a06.png)

Reply:
Then I would approve the PR ✔️